helpers.js

```diff
@@ -31,7 +35,45 @@ function logToActivityLog(message) {
   console.error(message);
 }
 
+async function waitForExecutionEnd({
+  pipelineId,
+  runId,
+  authToken,
+}) {
+  const serverUrl = await getServerUrl();
+  const apiUrl = `${serverUrl}/api/maps/${pipelineId}/results/${runId}/`;
+
+  let status;
+  try {
+    const { data } = await axios({
+      url: apiUrl,
+      method: "GET",
+      headers: {
+        Authorization: `Bearer ${authToken}`,
+      },
+      json: true,
+    });
+    status = data.status;
+  } catch (error) {
+    throw new Error(`Kaholo server threw an error: ${error.response.data}`);
+  }
+
+  if (status === STATUS_DONE) {
+    return status;
+  }
+
+  await delay(WAIT_INTERVAL_TIME);
```

Comment:
If you mean "interval of the wait time", then `WAIT_TIME_INTERVAL` is the gramatically corret version.
If you mean what's actually written there, then there's no need to specify `TIME` because `INTERVAL` always relates to `TIME`.